nexus/src/external_api/http_entrypoints.rs

```diff
@@ -269,6 +269,7 @@ pub fn external_api() -> NexusApiDescription {
         api.register(certificate_delete)?;
 
         api.register(system_metric)?;
+        api.register(silo_metric)?;
```

Comment:
@ahl what do you think about the operation names here? It's a little awkward to match our current naming scheme. Technically the metrics endpoint does return a list of metrics so we could potentially call it `system_metric_list`... but that makes it sound like it's listing out different _kinds_ of metrics. It would almost be more accurate to use `system_metric_view` in that you're viewing the system metrics... but it's a list.